packages/server/package.json

```diff
@@ -89,6 +89,7 @@
         "rimraf": "^5.0.5",
         "run-script-os": "^1.1.6",
         "shx": "^0.3.3",
+        "source-map-support": "^0.5.21",
```

Comment:
do we still need `source-map-support`? from looking at the repo [README](https://github.com/evanw/node-source-map-support#readme) it looks like we don't need it anymore?